bin/benchmark.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:math';
```

Comment:
This should be under a `benchmark` dir - `bin` is for public scripts (`pub run bazel_worker:benchmark`)